docs/model_server_rest_api.md

```diff
@@ -180,4 +183,103 @@ A request in [column format](https://www.tensorflow.org/tfx/serving/api_rest#spe
   "outputs": <value>|<(nested)list>|<object>
 }
 ```
-Read more about *Predict API* usage [here](./../example_client/README.md#predict-api-1)
\ No newline at end of file
+Read more about *Predict API* usage [here](./../example_client/README.md#predict-api-1)
+
+## Config Reload API <a name="config-reload"></a>
+* Description
+
+Sends requests via RESTful API to trigger config reloading and get models and DAGs statuses as a response.
+Flow after receiving request:
+1) If config file was changed - reload config. (If reload was needed return code is 201, otherwise 200)
```

Comment:
What if new version appeared, but config didn't change?  (what code is returned?)

Reply:
the reload was needed and successfully applied, the returned status code is `201`. When there are no config changed, status `200` is returned.